presto-product-tests/conf/docker/acid/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+  # Cannot put hadoop3 as a service in standard.yml and override depends_on in presto-master here to use hadoop3
+  # because depends_on list is append only, not overwritable
+  hadoop-master:
+    image: 'shubhamtagra/hdp3-hive3:1'
```

Comment:
I will open a PR for this in docker-images separately and replace it with prestosql image if this PR looks fine.

Reply:
Please do! I'm guessing this was one of the more difficult parts of this PR. I tried to make such an image a while ago but gave up. This will be very helpful for many of the Hive 3.x differences, including the bucketing and timestamp changes. Thanks making this work.

Reply:
Opened the PR in docker-images: https://github.com/prestosql/docker-images/pull/13

Once that is merged and a prestosql image is released, I will update this PR with new image as well as take care of your other comment about splitting this into two commits.

Reply:
I have updated this PR with the latest image built on top of changes in https://github.com/prestosql/docker-images/pull/13. This was suggested in docker-images PR.